src/graphql/data/nft/Details.ts

```diff
@@ -1,14 +1,12 @@
 import { parseEther } from '@ethersproject/units'
-import graphql from 'babel-plugin-relay/macro'
-import { CollectionInfoForAsset, GenieAsset, SellOrder } from 'nft/types'
-import { useEffect } from 'react'
-import { useLazyLoadQuery, useQueryLoader } from 'react-relay'
+import gql from 'graphql-tag'
+import { CollectionInfoForAsset, GenieAsset, Markets, SellOrder } from 'nft/types'
 
-import { NftStandard } from '../__generated__/types-and-hooks'
-import { DetailsQuery } from './__generated__/DetailsQuery.graphql'
+import { NftAsset, useDetailsQuery } from '../__generated__/types-and-hooks'
 
-const detailsQuery = graphql`
-  query DetailsQuery($address: String!, $tokenId: String!) {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const detailsQuery = gql`
```

Comment:
i saw in the other pr you got rid of the variable assignment? can you do it here?